HT-2.py

```diff
@@ -0,0 +1,67 @@
+# Підраховує кількість голосних літер у тексті
+
+
+def count_vowels(text):
+    vowels = 'AEIOUaeiou'
+    vowel_counts = {vowel: 0 for vowel in vowels}
+    for char in text:
+        if char in vowels:
+            vowel_counts[char] += 1
+
+    return vowel_counts
+
+# При запуску цього коду з використанням наданого тексту,
```

Comment:
Not critical. 
Better all comments leave in English.  You code could be read not only by Ukrainians ( common situation for many IT companies when offices are all round the world)